docs/user-guides/workspace-access/zed.md

```diff
@@ -0,0 +1,36 @@
+# Zed
+
+[Zed](https://zed.dev/) is an [open-source](https://github.com/zed-industries/zed)
+multiplayer code editor from the creators of Atom and Tree-sitter.
+
+## Use Zed to connect to Coder via SSH
+
+Use the Coder CLI to log in and configure SSH, then connect to your workspace with Zed:
+
+1. [Install Zed](https://zed.dev/docs/)
+1. Install Coder CLI:
+
+   ```shell
```

Comment:
Use tabs to list instructions for macOS/Linux and Windows
For Windows, you can use `winget install Coder.Coder`